containers/images/core/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM centos:7
```

Comment:
does it make sense to do a yum upgrade as the centos builds are monthly?

Reply:
I guess it makes sense if we want to setup a container with the latest changes and test it or validate it works. Nevertheless, if we want to provide `prod` image from `source` I don't think it's a good idea.
In addition, if we're going to do `yum upgrade` I believe it should go hand in hand with some kind of vulnerability static analysis such as `clair`.